Tests/NIOTransportServicesTests/NIOTSBootstrapTests.swift

```diff
@@ -246,6 +246,72 @@ final class NIOTSBootstrapTests: XCTestCase {
         XCTAssertNil(NIOTSListenerBootstrap(validatingGroup: wrongELG, childGroup: correctELG))
         XCTAssertNil(NIOTSListenerBootstrap(validatingGroup: wrongEL, childGroup: correctEL))
     }
+    
+    func testEndpointReuseShortcutOption() throws {
+        let group = NIOTSEventLoopGroup()
+        let listenerChannel = try NIOTSListenerBootstrap(group: group)
+            .bind(host: "127.0.0.1", port: 0)
+            .wait()
+        
+        var optionValue : EventLoopFuture<Bool>? = nil
+        let bootstrap = NIOClientTCPBootstrap(NIOTSConnectionBootstrap(group: group),
+                                              tls: NIOInsecureNoTLS())
+            .channelConvenienceOptions([.allowLocalEndpointReuse])
+            .channelInitializer { channel in
+                optionValue = channel.getOption(NIOTSChannelOptions.allowLocalEndpointReuse)
+                return channel.eventLoop.makeSucceededFuture(())
+            }
+        let client = try bootstrap.connect(to: listenerChannel.localAddress!).wait()
+        try client.close().wait()
+        
+        XCTAssertEqual(try optionValue!.wait(), true)
+    }
+    
+    func testShorthandOptionsAreEquvalent() throws {
+        func setAndGetOption<Option>(option: Option, _ applyOptions : (NIOClientTCPBootstrap) ->
+                NIOClientTCPBootstrap) throws -> Option.Value where Option : ChannelOption {
```

Comment:
Breaking the closure signature on two lines looks _really_ weird. I suggest a new line after `Option,` and vertically aligning with the argument above:

```
func setAndGet(option: Option,
               foo: Bar) ...
```

Reply:
I have done that.  The return part of function feels a little odd now - toyed with various indentations but none looked great.